searching for nodes after Shift-A -> error

On Macbook, Blender 3.0 in Sverchok version 1.0 with latest update i cannot search anymore for nodes, got this error:

![image](https://user-images.githubusercontent.com/232549/129960973-08cf11a3-4645-4cfe-bf7b-b6052dafb162.png)

I have this error but I can't reproduce it.
```
2021-09-08 15:57:42,647 [ERROR] sverchok.utils.sv_extra_search 68: Nodetype "<class 'sverchok.nodes.solid.bound_box.SvSolidBoundBoxNode'>": ensure_valid_show_string() threw an exception:
 'Node' object has no attribute 'docstring'
TypeError: EnumProperty(...): expected a tuple containing (identifier, name, description) and optionally an icon name and unique number
File "...\sverchok\utils\sv_extra_search.py", line 137, in item_cb
```

--- a/utils/sv_extra_search.py
+++ b/utils/sv_extra_search.py
@@ -184,7 +184,17 @@ class SvExtraSearch(bpy.types.Operator):
 
     def invoke(self, context, event):
         context.space_data.cursor_location_from_region(event.mouse_region_x, event.mouse_region_y)
-        loop['results'] = gather_items(context)
+        
+        if not loop.get('results'):
+            loop['results'] = gather_items(context)
+            print(":) extra search results cached")
+        else:
+            # line_1 = "using cached items! clear cache by doing\n"
+            # line_2 = ">>> import sverchok\n"
+            # line_3 = ">>> sverchok.utils.sv_extra_search.loop[\"results\"] = None"
+            # print(line_1 + line_2 + line_3)
+            pass
+        
         wm = context.window_manager
         wm.invoke_search_popup(self)
         return {'FINISHED'}

--- a/utils/sv_extra_search.py
+++ b/utils/sv_extra_search.py
@@ -187,13 +187,6 @@ class SvExtraSearch(bpy.types.Operator):
         
         if not loop.get('results'):
             loop['results'] = gather_items(context)
-            print(":) extra search results cached")
-        else:
-            # line_1 = "using cached items! clear cache by doing\n"
-            # line_2 = ">>> import sverchok\n"
-            # line_3 = ">>> sverchok.utils.sv_extra_search.loop[\"results\"] = None"
-            # print(line_1 + line_2 + line_3)
-            pass
         
         wm = context.window_manager
         wm.invoke_search_popup(self)

